examples/notebook/src/commands.ts

```diff
@@ -164,11 +166,19 @@ export const SetupCommands = (
   });
   commands.addCommand(cmdIds.extendAbove, {
     label: 'Extend Above',
-    execute: () => NotebookActions.extendSelectionAbove(nbWidget.content)
+    execute: () => NotebookActions.extendSelectionAbove(nbWidget.content, false)
+  });
+  commands.addCommand(cmdIds.extendTop, {
+    label: 'Extend to Top',
+    execute: () => NotebookActions.extendSelectionAbove(nbWidget.content, true)
   });
   commands.addCommand(cmdIds.extendBelow, {
     label: 'Extend Below',
-    execute: () => NotebookActions.extendSelectionBelow(nbWidget.content)
+    execute: () => NotebookActions.extendSelectionBelow(nbWidget.content, false)
```

Comment:
```suggestion
    execute: () => NotebookActions.extendSelectionBelow(nbWidget.content)
```

Reply:
I have updated the file with the above changes.